viewer/packages/rendering/src/glsl/sector/instancedMesh.frag

```diff
@@ -13,11 +13,14 @@ uniform sampler2D matCapTexture;
 uniform vec2 treeIndexTextureSize;
 uniform int renderMode;
 
-flat in float v_treeIndex;
 in vec3 v_color;
 in vec3 v_viewPosition;
 
-void main() {
+in highp float v_treeIndexHundreds;
+in mediump float v_treeIndexSubHundreds;
```

Comment:
Suggestion: change this to `in highp vec2 v_treeIndex` (or possibly a struct, if you want to be more explicit), and create functions `packTreeIndex` and `unpackTreeIndex` or something and add some comments to the functions. 

`mediump` most often boils down to `highp` these days, as seen on https://webglfundamentals.org/webgl/webgl-precision-check-vertex-shader.html, so there is little gain on having a mediump for subhundres.